client/UI/FFRKViewInventory.cs

```diff
@@ -304,24 +334,52 @@ private void UpdatePartyGrid(List<DataBuddyInformation> buddies)
 
         void UpdateEquipmentGrid(DataEquipmentInformation[] EquipList)
         {
+            bool needFilter = true;
+            if (comboBoxFilterType.SelectedIndex.Equals(0)) { needFilter = false; }

```

Comment:
0 seems like a magic number and makes the code hard to understand, maybe you can make an enum similar to how i did with `ViewUpgradeModeComboIndex`